packages/reactor-api/src/server.ts

```diff
@@ -1,18 +1,11 @@
 import { PGlite } from "@electric-sql/pglite";
-import { AnalyticsQueryEngine } from "@powerhousedao/analytics-engine-core";
-import { AnalyticsModel } from "@powerhousedao/analytics-engine-graphql";
-import {
-  KnexAnalyticsStore,
-  KnexQueryExecutor,
-} from "@powerhousedao/analytics-engine-knex";
 import { IDocumentDriveServer } from "document-drive";
 import express, { Express } from "express";
 import { Pool } from "pg";
-import { ReactorRouterManager } from "./router";
-import { getKnexClient } from "./utils/get-knex-client";
-import { ProcessorManager } from "./processor-manager";
+import { ProcessorManager } from "./processors";
+import { SubgraphManager } from "./subgraphs/manager";
 import { API } from "./types";
-import { initialize } from "./subgraphs/analytics";
+import { getKnexClient } from "./utils/get-knex-client";
```

Comment:
Here could be getDbClient, and only `utils/db` "knows" about knex. If in the future we decide to use something else we just need to change `utils/db`.